zero/src/bin/leader/cli.rs

```diff
@@ -43,6 +43,9 @@ pub(crate) enum Command {
         // The node RPC type (jerigon / native).
         #[arg(long, short = 't', default_value = "jerigon")]
         rpc_type: RpcType,
+        /// The source of jumpdest tables.
+        #[arg(short = 'j', long, default_value_ifs = [("rpc_type", "jerigon", "client-fetched-structlogs"), ("rpc_type", "native", "client-fetched-structlogs")], required = false)]
```

Comment:
Two things:
- `jerigon` / `native` use the same value -> use `default` instead of `default_value_ifs`
- I think it's been mentioned on slack by @atanmarko or @0xaatif but this `client-fetched-structlogs` variant looks fairly cumbersome / tedious to have as CLI argument, would deserve a clearer + simpler name

Reply:
1. Done. 
2. Please share your suggestions @Nashtare @atanmarko @0xaatif